assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -1003,6 +1023,16 @@ public String buildMainPanelContext(VelocityPortlet portlet, Context context, Ru
 			context.put("reviewIndicator", rb.getFormattedMessage("review.contentReviewIndicator", new Object[]{reviewServiceName}));
 			context.put("reviewSwitchNe1", reviewServiceNonElectronic1);
 			context.put("reviewSwitchNe2", reviewServiceNonElectronic2);
+			if(contentReviewSiteAdvisor.siteCanUseLTIReviewService(s)){
+				allowLTIReviewService = true;
+			} else {
+				allowLTIReviewService = false;
+			}
+			if(contentReviewService.isDirectAccess(s)){
+				isDirectAccess = true;
+			} else {
+				isDirectAccess = false;
+			}
```

Comment:
Cut this down to:
allowLTIReviewService = contentReviewSiteAdvisor.siteCanUseLTIReviewService(s);
isDirectAccess = conrentReviewService.isDirectAccess(s);